### PR TITLE
[Test] add test for common-config

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: jimin.jm <slievrly@163.com>

## What is the purpose of the change

[Test] add test for EnvironmentConfiguration、InmemoryConfiguration、SystemConfiguration

## Brief changelog

[Test] add test for common-config

## Verifying this change

CI

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [x] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [x] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package org.apache.dubbo.common.config;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
```
Comment (Moriadry) [inline dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java]: don't record author in dubbo

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        Assertions.assertEquals(MOCK_VALUE,environmentConfig.getString(MOCK_KEY,MOCK_VALUE));
+    }
+
+    @Test
+    public void testGetProperty(){
+        Assertions.assertNull(environmentConfig.getProperty(MOCK_KEY));
+        Assertions.assertEquals(MOCK_VALUE,environmentConfig.getProperty(MOCK_KEY,MOCK_VALUE));
+    }
+
+
```
Comment (Moriadry) [inline dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java]: pls make this one single line.

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/InmemoryConfigurationTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author slievrly
```
Comment (Moriadry) [inline dubbo-common/src/test/java/org/apache/dubbo/common/config/InmemoryConfigurationTest.java]: remove this.

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
+package org.apache.dubbo.common.config;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author slievrly
```
Comment (Moriadry) [inline dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java]: another useless lines.

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        System.setProperty(MOCK_KEY, String.valueOf(ConfigMock.MockOne));
+        Assertions.assertEquals(ConfigMock.MockOne, sysConfig.convert(ConfigMock.class, MOCK_KEY, null));
+    }
+
+    enum ConfigMock {
+        MockOne,
+        MockTwo
+    }
+
+
```
Comment (Moriadry) [inline dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java]: make this one single line.

**Comment by slievrly** [comment]
@Moriadry all fixed.

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java (hunk 2 above)
Comment (slievrly) [reply dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java]: fixed

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/InmemoryConfigurationTest.java (hunk 3 above)
Comment (slievrly) [reply dubbo-common/src/test/java/org/apache/dubbo/common/config/InmemoryConfigurationTest.java]: fixed

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java (hunk 4 above)
Comment (slievrly) [reply dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java]: fixed

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java (hunk 5 above)
Comment (slievrly) [reply dubbo-common/src/test/java/org/apache/dubbo/common/config/SystemConfigurationTest.java]: fixed

File: dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java (hunk 1 above)
Comment (slievrly) [reply dubbo-common/src/test/java/org/apache/dubbo/common/config/EnvironmentConfigurationTest.java]: fixed

**Pull request merged**